examples/playground/src/app.js

```diff
@@ -18,6 +19,7 @@ const INITIAL_TEMPLATE = Object.keys(JSON_TEMPLATES)[0];
 
 // Set your mapbox token here
 const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
+const GOOGLE_MAPS_TOKEN = process.env.googleMapsApiKey; // eslint-disable-line
```

Comment:
Should we capitalize Google for consistency?

Reply:
Given that the Mapbox env variable is `MapboxAccessToken`, I'd be in favor of `GoogleMapsAccessToken` for consistency.